src/qml/LoginPage.qml

```diff
@@ -2,37 +2,38 @@ import QtQuick 2.5
 import QtQuick.Controls 1.2
 import harbour.kaidan 1.0
 
-Item{
-	Column{
-		Row{
-			TextField{
+Item {
+	Column {
+		Row {
+			TextField {
 				id: jidField
 				width: 150
 				placeholderText: "JID"
 			}
-			TextField{
+			TextField {
 				id: passField
 				width: 150
 				height: 27
 				placeholderText: "Password"
 				echoMode: TextInput.Password
 			}
 		}
-		Button{
+		Button {
 			id: connectButton
 			text: "Connect"
-			onClicked:{
+			onClicked: {
 				connectButton.enabled = false;
 				kaidan.mainConnect(jidField.text, passField.text);
 			}
 		}
-		Label{
+		Label {
 			id: statusLabel
 			text: "Not connected"
 		}
 	}
-	Component.onCompleted:{
-		function goToRoster(){
+	Component.onCompleted: {
+		function goToRoster()
+		{
```

Comment:
I have to say in JS, I like this style more (as in #23 )
```javascript
function goToRoster() {
```

And please add a blank (or a new line) for the function in line 41/42.